### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by zoeames**

### CODE REVIEW

**Comment by zoeames** [comment]
@Sanyyouisf make sure you open a PR when you are ready for things to be reviewed.  Anything listed as challenge will be graded by an instructor not your big sib

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	<label for="height"><h3>the height of the tree</h3></label>
+	<input type="text" name="height" id="height" style="width: 40%" placeholder="How tall should the tree be?">
+
+<!-- A key that specifies which character to use to build the pine tree. -->
+<!-- The character to use should be from user input in a <input type="text"> field in the DOM. -->
+	<label for="character"><h3>Enter the character</h3></label>
+	<input type="text" name="character" id="character" style="width: 40%" placeholder="Which character should be used?"><br>
+
+	<label for="build"><h3>Grow your tree</h3></label>
+	<input type="button" name="build" id="build" >
```
Comment (zoeames) [inline index.html]: Use an actual Button tag here < button id="build" >Grow Your Tree< /button >

File: main.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+	// function when push the button 
+	function height(height,space){
```
Comment (zoeames) [inline main.js]: you you really need the words height, space on here?

**Comment by zoeames** [comment]
I should also be able to grow a tree by clicking the enter button.  Make sure you add that functionality and make the 2 changes I suggested above

**Comment by Sanyyouisf** [comment]
finish

**Comment by zoeames** [comment]
Looks good! 👍  I will merge for you

**Pull request merged**